Sources/App/Extensions/ArrayExtensions.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ArrayExtensions.swift
+//  App
+//
+//  Created by Tiewhan Smith on 2020/04/20.
+//
+
+import Foundation
+
+extension Array {
+  
+  public func convertByteArrayToJSON(withData data: [UInt8]) -> String {
+    
+    var jsonString = "{ \"data\" : [ "
+    var isFirstIteration = true
+    
+    data.forEach { number in
```

Comment:
Instead of this, you could map your data array into a string array. Then simply used the array.joinedWithSeperator Function to add your commas without needed to check if it's the first item or anything